test/test.js

```diff
@@ -2847,6 +2847,15 @@ describe('external encoding', () => {
 });
 
 describe('issue #1290', function() {
+
+	it('should keep query params', function() {
+		return fetch(`${base}inspect?month=2021-09`)
+		  .then(res => res.json())
+			.then(json => {
+				expect(json.url).to.equal('/inspect?month=2021-09')
+			})
+	})
+
```

Comment:
thanks for putting in a test to avoid future regressions 💐 

This explains the mystery of https://github.com/octokit/rest.js/issues/114 and https://github.com/semantic-release/github/issues/415